docs/migration-case.md

```diff
@@ -0,0 +1,44 @@
+# Cluster migration
+
+*Using Backups and Restores*
+
+Heptio Ark can help you port your resources from one cluster to another, as long as you point each Ark instance to the same cloud object storage location. In this scenario, we are also assuming that your clusters are hosted by the same cloud provider. **Note that Heptio Ark does not support the migration of persistent volumes across cloud providers.**
+
+1.  *(Cluster 1)* Assuming you haven't already been checkpointing your data with the Ark `schedule` operation, you need to first back up your entire cluster (replacing `<BACKUP-NAME>` as desired):
+
+    ```
+    ark backup create <BACKUP-NAME>
+    ```
+    The default TTL is 30 days (720 hours); you can use the `--ttl` flag to change this as necessary.
+
+1.  *(Cluster 2)* Make sure that the `BackupStorageLocation` and `VolumeSnapshotLocation` CRDs match the ones from *Cluster 1*, so that your new Ark server instance points to the same bucket.
```

Comment:
we should also have them put cluster 2 in restore-only mode (using the server flag) for this

Reply:
I had always assumed (and apparently never communicated) that "restore only mode" would move from the `Config` to each `BackupStorageLocation` and that you could enable/disable with updates to a BSL. How do you feel about that?

Reply:
Yeah, we were planning to do that in v0.11 as part of the ownership/sharing epic

Reply:
Since we're pausing on the replication train though, we'll need to decide if we want to do any parts of those epics for v1.0.  They weren't in our issue list since we hadn't broken them out yet.

Reply:
Makes sense, thanks